tools/gen_ort_dockerfile.py

```diff
@@ -206,23 +221,23 @@ def dockerfile_for_linux(output_file):
        /opt/onnxruntime/include
 
 RUN mkdir -p /opt/onnxruntime/lib && \
-    cp /workspace/build/Release/libonnxruntime_providers_shared.so \
+    cp /workspace/build/${ONNXRUNTIME_BUILD_CONFIG}/libonnxruntime_providers_shared.so \
```

Comment:
Does this PR need to be rebased? It seems to have the debug build changes. Can we merge the debug changes and rebase this? Or are debug changes not ready yet?

Reply:
Rebase Done.